src/main/java/com/github/instagram4j/instagram4j/requests/search/SearchLocationRequest.java

```diff
@@ -0,0 +1,62 @@
+package com.github.instagram4j.instagram4j.requests.search;
```

Comment:
Naming conventions for requests adhere to a naming scheme based on the requests name.

For example, for fbsearch/places,

This should be placed under fbsearch package under the requests package.

Then the request class name should be FbsearchPlacesRequest 

The same naming convention and package name should also apply to response objects as well.

(FbsearchPlacesResponse should be placed under the "fbsearch" package under the responses package.